.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
```

Comment:
Question: I am wondering if we should (and if we can) add `anthos-dpe` as an assignee to all 3 of these templates?

Reply:
hmm i was initially thinking that, but i was also weary of the potential notification overload 😅  we could just add it for the time being, and see how it works?

Reply:
> potential notification overload

Ah, trueee! You're right. Plus we always have someone on rotation monitoring the dashboard. 
Let's not add assignees.